websocket/connection.go

```diff
@@ -120,11 +124,12 @@ func NewDurableConnection(target string, messageChan chan []byte) *ManagedConnec
 }
 
 // newConnection creates a new connection primitive.
-func newConnection(target string, messageChan chan []byte) *ManagedConnection {
+func newConnection(target string, messageChan chan []byte, connFactory func(target string) (rawConnection, error)) *ManagedConnection {
 	conn := &ManagedConnection{
-		target:      target,
-		closeChan:   make(chan struct{}, 1),
-		messageChan: messageChan,
+		target:            target,
+		connectionFactory: connFactory,
+		closeChan:         make(chan struct{}),
+		messageChan:       messageChan,
 		connectionBackoff: wait.Backoff{
 			Duration: 100 * time.Millisecond,
```

Comment:
Can we comment why did we pick those exact values?

Reply:
I can't really. These have been based off what the hardcoded backoff was in the queue proxy before + some spreadsheeding to find a good fit here. The hockeystick seemed about right for this purpose.